### ISSUE TITLE
Samples failed to start due to ConfigurationException

### ISSUE THREAD

**Issue by TKYAngus** [real-logic/artio#504]
When trying to start the sample applications (e.g. ExchangeApplication, SampleClient, SampleServer, etc.) I always got exceptions, e.g.:
```
Exception in thread "main" io.aeron.exceptions.ConfigurationException: ERROR - Archive.Context.controlChannel must be set
```

```
Exception in thread "main" io.aeron.exceptions.ConfigurationException: ERROR - Archive.Context.replicationChannel must be set
```

Can I know are those configs required? If so I think we should fix the sample codes

**Comment by martin-nginio** [comment]
@RichardWarburton and @wojciech-adaptive 

any help getting this sample working would be appreciated.

**Comment by 0x26res** [comment]
I got SampleClient/SampleClient to work with the following changes https://github.com/real-logic/artio/pull/516

**Comment by 0x26res** [comment]
This is fixed in master with 09b8b13fb247484f16acebd3515bfe73388c46f6 though I haven't tested ExchangeApplication.

**Comment by martin-nginio** [comment]
thanks @0x26res

**Comment by marc-adaptive** [comment]
These should be fixed in 09b8b13fb247484f16acebd3515bfe73388c46f6 and e2891587765af8d2505b3f6e00599ac180bc2a96. Please let us know if you are still having problems.
